### DISCUSSION TITLE
Usage of RCT HA plugin in combination with modbus proxy

### DISCUSSION THREAD

**Discussion by diomedon** [weltenwort/home-assistant-rct-power-integration#309] [Q&A]
Hi,
Does anyone use the HA RCT integration together with a modbus proxy? I neither get this plugin nor iobroker nor evcc to communicate with my RCT inverter via a modbus proxy. I have used this HA plugin for example https://github.com/Akulatraxas/ha-modbusproxy

Regards 
Carsten

**Comment by weltenwort** [comment]
The communication protocol that RCT uses is not modbus, but seems to be a proprietary serial-over-tcp "invention". So I doubt a modbus proxy could be of any use here.

**Reply by diomedon** [reply]
ahh, therefore it does not work. :-( thanks
